doc/Analytics.xml

```diff
@@ -6539,6 +6556,61 @@
               <entry>
                 <para>Optional</para>
               </entry>
+            </row>
+			<row>
+              <entry>
+                <para>Country</para>
+              </entry>
+              <entry>
+                <para>Specifies the country for which the LPR should be customized, if the country is not specified, Generic mode will be used.</para>
+              </entry>
+              <entry>
+                <para>Optional</para>
+              </entry>
+            </row>
+			<row>
+              <entry>
+                <para>Sensitivity</para>
+              </entry>
+              <entry>
+                <para>detection sensitivity which provides compromise between detection rate & false alarms.Higher sensitivity level more License plate detected at the cost of increasing false alarms number.</para>
+              </entry>
+              <entry>
+                <para>Optional</para>
+              </entry>
+            </row>
+			<row>
+              <entry>
+                <para>PlateMinWidth</para>
+              </entry>
+              <entry>
+                <para>Minimum width of the license plate in percentage to be detected/recognized.</para>
```

Comment:
It would be better to mention this width in percentage is relative to the Scene Image/Full Frame.